content/lessons/chapter-4/address-2.tsx

```diff
@@ -29,7 +29,9 @@ export default function Address2({ lang }) {
   }, [])
 
   const javascript = {
-    program: `console.log("KILL")`,
+    program: `
```

Comment:
We need to update the solution in `address.tsx` in resources to reflect this change.